### PR TITLE
Support Python 3.6 and 3.7; discourage usage of 3.4

### PR DESCRIPTION
**Pull request by michael-k**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,4 +12,22 @@
     test_suite="tests",
     use_2to3=True,
     requires=['slimit'],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
```
Comment (michael-k) [inline setup.py]: I did not add 3.4 here for consistency with troposphere.  But by now it's past its EOL and I'd love to add it as well.

File: setup.py [hunk 2]
```diff
@@ -12,4 +12,22 @@
...
+    classifiers=[
```
Comment (markpeek) [inline setup.py]: Last time I pushed a classifier change (to troposphere) it broke PyPI and I applied this [change](https://github.com/cloudtools/troposphere/commit/47138aa1928aa759a032271801eb105f7c9d7c5e). I believe your change matches the fixed version.

Comment (michael-k) [reply setup.py]: Yeah, I think I copied it from troposphere.

File: setup.py (hunk 1 above)
Comment (michael-k) [reply setup.py]: Done

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**